installation/standard.py

```diff
@@ -48,8 +48,8 @@
 #     If you set x11libdir (that is two ones and an el) AND x11include to point
 #     to the lib and include directories, they will be used.
 #     Otherwise a search is made for common locations.
-x11libdir=''
-x11include=''
+x11libdir='/usr/X11R6/lib'
+x11include='/usr/X11R6/include'
```

Comment:
@jbeezley all of the modern Apple apps don'e use X11 and infact its not preferred. We only have one GUI app and it should be not be using X11 either. Unless we really need X11, I don't think making it a hard requirement is good in my opinion. We should use native libraries as much as possible.